lnwallet/channel.go

```diff
@@ -350,6 +352,7 @@ func (c *commitment) toChannelDelta(ourCommit bool) (*channeldb.ChannelDelta, er
 			RefundTimeout:   htlc.Timeout,
 			RevocationDelay: 0,
 			OutputIndex:     index,
+			OnionBlob:       htlc.OnionBlob,
```

Comment:
Where's the matching `channeldb` commit to ensure that this state is properly written to disk?